src/app/components/map/map.controller.js

```diff
@@ -160,9 +160,11 @@
 
         vm.markers = device.getWorldMarkers();
 
-        device.getAllDevices()
+        $q.all([device.getAllDevices(), device.createKitBlueprints()])
```

Comment:
I know why, because I looked at the service, but `device.createKitBlueprints()` should be maybe in a resolver in the routes ?

Reply:
I merge this to develop but I open myself an issue to do a whole refactoring for `createKitBlueprints` and other services that need to be there most of the time